### DISCUSSION TITLE
Restoring Möbius purchase from a new device / redownload

### DISCUSSION THREAD

**Discussion by MrQvest** [MobiusSync/MobiusSync#200] [Q&A] [answered]
I purchased Möbius through the in-app purchase method. 

I used a temporary digital card to do it (meaning one that would be deleted in 24h).

How could I restore this purchase in a new iPhone or if I accidentally delete Möbius? I have an Apple ID login.

**Comment by SimonPickup** [comment, marked as answer]
As long as you are logged in with the same Apple ID, you can restore purchases using the Restore Purchases button under menu/Purchase. The details of the card you used should be irrelevant, but that's between you and Apple - developers don't get to see such detail.
